glossary.rst

```diff
@@ -0,0 +1,67 @@
+Glossary
+========
+
+.. glossary::
+
+    Application
+        Website or application tracked by PPMS.
+
+    App ID
+        PPMS :term:`application` identificator (previously **website ID**, **site ID** or **idSite**).
+
+    User
+        Visitor on tracked :term:`application`.
+
+    Analytics ID
+        ID assigned to :term:`user` by :term:`Analytics` for the duration of :term:`Analytics` session. It's stored in
+        browser cookie.
+
+    User ID
+        Permanent ID assigned to :term:`user` by :term:`application` (e.g. username). You can read more about it
```

Comment:
"(...) assigned to user by application (...)" is somewhat misleading. User ID is an external ID that only gets passed to the application.

Reply:
Application is defined in another part of glossary as a client app using PPMS for tracking, not PPMS itself. Does that make it clear?